actionmailbox/app/controllers/rails/conductor/action_mailbox/inbound_emails_controller.rb

```diff
@@ -20,7 +20,7 @@ def create
 
     private
       def new_mail
-        Mail.new(params.require(:mail).permit(:from, :to, :cc, :bcc, :in_reply_to, :subject, :body).to_h).tap do |mail|
+        Mail.new(params.require(:mail).permit(:from, :to, :cc, :bcc, :in_reply_to, :subject, :body, attachments: []).to_h).tap do |mail|
           mail[:bcc]&.include_in_headers = true
           params[:mail][:attachments].to_a.each do |attachment|
```

Comment:
The test didn't fail because we're looping over the attachments here, so this code works already.

Though it doesn't support having `action_on_unpermitted_parameters = :raise` (or what the config is called) set in development. Is that config enabled in your app? Fine with this code adapting to support it. (That config is generally not set to `:raise` in development because unpermitted parameters aren't an error state exactly like this action is a show case of.)

Reply:
Hi @kaspth, that's correct. This development app has `action_on_unpermitted_parameters` set to `:raise` in the `application.rb`. It stems from [thoughtbot/suspenders](https://github.com/thoughtbot/suspenders), and as you say, not usually set in development.